js/geo/transform.js

```diff
@@ -260,6 +260,16 @@ Transform.prototype = {
     },
 
     /**
+     * Like `locationPoint` but returns a raw 4d vector containing info about the transform
+     * @param {LngLat} lnglat location
+     * @returns {Array} The 4d vector pertaining to this point's transform
+     * @private
+     */
+    locationPoint3d: function(lnglat) {
+        return this.coordinatePoint3d(this.locationCoordinate(lnglat));
+    },
```

Comment:
My impression is that that the only difference between `locationPoint3d` and `locationPoint` is an extra property object returned from `locationPoint3d`. Is that not the case?